### PR TITLE
OADP-927: Remove VSClass name in Get error

### PR DESCRIPTION
**Pull request by eemcmullan**

### CODE REVIEW

File: pkg/datamover/datamover.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func DeleteTempVSClass(backupName string, tempVS snapshotv1listers.VolumeSnapsho
 
 	tempVSClass, err := tempVS.Get(fmt.Sprintf("%s-snapclass", backupName))
 	if err != nil {
-		log.Errorf("failed to get temp vsClass %v", tempVSClass.Name)
+		log.Error("failed to get temp vsClass for vsb")
 		return err
 	}
```
Comment (shubham-pampattiwar) [inline pkg/datamover/datamover.go:143]: Add a nil check for `tempVSClass` before executing the Delete call on line 143?

File: pkg/datamover/datamover.go [hunk 2]
```diff
@@ -136,7 +136,7 @@ func DeleteTempVSClass(backupName string, tempVS snapshotv1listers.VolumeSnapsho
 
 	tempVSClass, err := tempVS.Get(fmt.Sprintf("%s-snapclass", backupName))
 	if err != nil {
-		log.Errorf("failed to get temp vsClass %v", tempVSClass.Name)
```
Comment (shubham-pampattiwar) [inline pkg/datamover/datamover.go:139]: Maybe you frame the tempVSClass Name first in a local var, that way you can use that var in the log to specify the vsclass name ?

Comment (eemcmullan) [reply pkg/datamover/datamover.go:139]: The issue is this temp VSClass does not get created if VSB fails early, so assigning a var to this name would be nil in this case, which then throws an error.

Comment (shubham-pampattiwar) [reply pkg/datamover/datamover.go:139]: So what I meant was use,
`tempVSClassName := fmt.Sprintf("%s-snapclass", backupName)` then use `tempVSClassName` in the logs. We should be having the backupName and `tempVSClassName` is constructed per backup using `sprintf`

File: pkg/datamover/datamover.go [hunk 3]
```diff
@@ -134,16 +134,20 @@ func WaitForDataMoverBackupToComplete(backupName string) error {
 
 func DeleteTempVSClass(backupName string, tempVS snapshotv1listers.VolumeSnapshotClassLister, client *snapshotterClientSet.Clientset) error {
 
+	tempVSClassName := fmt.Sprintf("%s-snapclass", backupName)
 	tempVSClass, err := tempVS.Get(fmt.Sprintf("%s-snapclass", backupName))
```
Comment (shubham-pampattiwar) [inline pkg/datamover/datamover.go]: Shouldn't we directly use the `tempVSClassName` in the `Get` call ?

**Review by shubham-pampattiwar — APPROVED** [review]
Thank you @eemcmullan lgtm!

**Pull request merged**